public/app/plugins/datasource/mssql/configuration/ConfigurationEditor.tsx

```diff
@@ -123,46 +141,55 @@ export const ConfigurationEditor = (props: DataSourcePluginOptionsEditorProps<Ms
                 <i>Windows Authentication</i> Windows Integrated Security - single sign on for users who are already
                 logged onto Windows and have enabled this option for MS SQL Server.
               </li>
+              {azureAuthIsSupported && (
+                <li>
+                  <i>Azure Authentication</i> Securely authenticate and access Azure resources and applications using
+                  Azure AD credentials - Managed Service Identity and Client Secret Credentials are supported.
+                </li>
+              )}
             </ul>
           }
         >
           <Select
+            // Default to basic authentication of none is set
             value={jsonData.authenticationType || MSSQLAuthenticationType.sqlAuth}
             inputId="authenticationType"
-            options={authenticationOptions}
+            options={buildAuthenticationOptions()}
             onChange={onAuthenticationMethodChanged}
           ></Select>
         </InlineField>
-        {jsonData.authenticationType === MSSQLAuthenticationType.windowsAuth ? null : (
-          <InlineFieldRow>
-            <InlineField labelWidth={shortWidth} label="User">
-              <Input
-                width={shortWidth}
-                value={options.user || ''}
-                placeholder="user"
-                onChange={onDSOptionChanged('user')}
-              ></Input>
-            </InlineField>
-            <InlineField label="Password" labelWidth={shortWidth}>
-              <SecretInput
-                width={shortWidth}
-                placeholder="Password"
-                isConfigured={options.secureJsonFields && options.secureJsonFields.password}
-                onReset={onResetPassword}
-                onBlur={onUpdateDatasourceSecureJsonDataOption(props, 'password')}
-              ></SecretInput>
-            </InlineField>
-          </InlineFieldRow>
-        )}
+        {/* Basic SQL auth */}
+        {jsonData.authenticationType === MSSQLAuthenticationType.sqlAuth ||
+          (jsonData.authenticationType === MSSQLAuthenticationType.windowsAuth && (
```

Comment:
This logic is not correct. The user and password fields should only be displayed when SQL Server Authentication is selected. When windows authentication is selected credentials will come from windows single sign on.

Reply:
Updated.